Source/Events.Handling/Builder/EventHandlerMethod.cs

```diff
@@ -11,13 +12,25 @@ namespace Dolittle.SDK.Events.Handling.Builder
     /// </summary>
     public class EventHandlerMethod : IEventHandlerMethod
     {
-        readonly EventHandlerSignature _method;
+        readonly Func<object, EventContext, Task> _method;
 
         /// <summary>
         /// Initializes a new instance of the <see cref="EventHandlerMethod"/> class.
         /// </summary>
         /// <param name="method">The <see cref="EventHandlerSignature" />.</param>
-        public EventHandlerMethod(EventHandlerSignature method) => _method = method;
+        public EventHandlerMethod(EventHandlerSignature method)
+            => _method = (@event, context) => method(@event, context);
```

Comment:
Why do we need to create a lambda here?

Reply:
This should be resolved by latest commits